moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -433,7 +433,12 @@ void MotionPlanningFrame::changePlanningGroup()
 void MotionPlanningFrame::sceneUpdate(planning_scene_monitor::PlanningSceneMonitor::SceneUpdateType update_type)
 {
   if (update_type & planning_scene_monitor::PlanningSceneMonitor::UPDATE_GEOMETRY)
```

Comment:
Should this be `&&`?

Reply:
No, I think that's meant to be a bitwise AND. It compares `update_type` with the enum value `...UPDATE_GEOMETRY`. Would be easier to read if it was `==` imo, but this works so I don't see a need to change it.